public/wp-content/themes/ColoredCow/header.php

```diff
@@ -7,6 +7,25 @@
     <meta name="viewport" content="initial-scale=1">
     <title><?php echo bloginfo('name'); ?></title>
     <?php wp_head(); ?>
+
+	<nav class=" navbar fixed-top navbar-toggleable-md navbar-inverse bg-inverse">				
+		<button class="navbar-toggler navbar-toggler-right justify-content-end" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+			<span class="navbar-toggler-icon"></span>
+		</button>
+		<a class="navbar-brand" href="#">
+			<img src="<?php echo get_template_directory_uri(); ?>/dist/img/logo.png" width="180" height="45" class="d-inline-block align-top" alt="">
```

Comment:
let's use width and height in the stylesheet. This also works but let's keep all the styling and layout in the stylesheet.